### ISSUE TITLE
Error when one display is dpms off

### ISSUE THREAD

**Issue by kamiyaa** [emersion/grim#81]
Not sure if this is the expected behavior but if one display is turned off via dpms (ie. a laptop screen)
grim errors out with (in this case `eDP-1`):
```
failed to copy output eDP-1
```
I would think grim would just grab the other existing display(s)

**Comment by emersion** [comment]
Right. I wonder if wlroots should just fill a black frame in this case. Or go through requesting a new frame for all visible clients on the DMPS'ed output, then capture.
